src/main/java/mate/academy/springbootwebgreqit/service/impl/BookServiceImpl.java

```diff
@@ -51,5 +55,17 @@ public BookDto update(UpdateBookRequestDto requestDto) {
         bookMapper.updateBookFromDto(requestDto, book);
         Book updatedBook = bookRepository.save(book);
         return bookMapper.toDto(updatedBook);
+
```

Comment:
remove blank lines

Reply:
I don't know why, but program doesn't start
![Uploading Screenshot 2024-07-03 at 12.37.27.png…]()

Reply:
It's not related to blank lines)))